libraries/botbuilder-dialogs-adaptive/src/converters/recognizerConverter.ts

```diff
@@ -10,28 +10,28 @@ import { Converter } from 'botbuilder-dialogs';
 import { ResourceExplorer } from 'botbuilder-dialogs-declarative';
 import { Recognizer } from '../recognizers';
 
-export class RecognizerConverter implements Converter<any, Recognizer> {
+export class RecognizerConverter implements Converter<unknown, Recognizer> {
```

Comment:
This should be `Converter<string | unknown, Recognizer>` to match the `convert` method signature.